src/scipp/core/bins.py

```diff
@@ -228,6 +228,11 @@ def unit(self, unit: Union[_cpp.Unit, str]):
         """Set unit of the bin elements"""
         self.constituents['data'].unit = unit
 
+    @property
+    def aligned(self) -> bool:
+        """Alignment flag for coordinates."""
```

Comment:
Might be worth adding an example, so show the non-obvious `da.bins.coords['x'].bins.aligned` syntax?

Reply:
Where? Adding it here requires that people already know about this.

Reply:
I thought here, in the docstring. Otherwise you get it as auto-complete suggestion, and users will wonder why `da.bins.coords['x'].aligned` does not work.

Reply:
Well, people have to know that `da.bins.coords['x'].aligned` is always true. So it should be in the docstring of `Variable.aligned`

Reply:
Added an explanation in the docs of `Variable.aligned`. We should also add something in the user guide